src/main/java/uk/gov/hmcts/darts/common/entity/RetentionConfidenceCategoryMapperEntity.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.darts.common.entity.base.MandatoryCreatedModifiedBaseEntity;
+
+@Entity
+@Table(name = "retention_confidence_category_mapper")
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class RetentionConfidenceCategoryMapperEntity extends MandatoryCreatedModifiedBaseEntity {
```

Comment:
This table seems to be a standalone table as there are no other tables that have a foreign key to this but this table also has no foreign keys to other tables. Am I missing something here?

Reply:
I believe that is indeed the case. However I don't see any design details on this table in confluence, the source of truth here is `src/main/resources/db/reference/retention.sql`.

Reply:
This is a placeholder table for now. The values are yet to be discussed. The RCC LLD will be updated once the mapping values are finalised.